activiti-cloud-service-common/activiti-cloud-services-dbp-rest/src/main/java/org/activiti/cloud/alfresco/argument/resolver/AlfrescoPageRequest.java

```diff
@@ -82,6 +82,11 @@ public AlfrescoPageRequest first() {
                                        pageSize, getPageable());
     }
 
+    @Override
+    public Pageable withPage(int pageNumber) {
+        return pageable.withPage(pageNumber);
```

Comment:
Actually, after double-checking, what you need to set here is the `skipCount` with `pageNumber * getPageSize()` because `getPageNumber` is overridden [here](https://github.com/Activiti/activiti-cloud/pull/455/files#diff-11ec11e53366eb734a751267e5ae0caea469ef247150dcab759642f9b2c891a1R51). Please, add a new test to  `AlfrescoPageRequestTest` to cover this.

Reply:
Fixed as you suggested, thanks.